src/views/articles.hbs

```diff
@@ -1,17 +1,23 @@
 
 {{>searchbar}}
 
-<div id="searchResults" class="search__results">
+<div id="searchResults"
+    class="search__results mw7 center">
   {{#each articles}}
-    <img src={{image}}></img>
-    <h3>{{headline}}</h3>
-    <p>{{trailText}}</p>
-      <form class"article_form" action="/visualize" method="POST">
+  <div class="bg-white pa4 center br3 mb4">
+    <div class="mw9 measure center">
+    <img class="br2" src={{image}}></img>
+    </div>
+    <div class="measure center">
+    <h3 class="f4 f2-ns mt2 mb3">{{headline}}</h3>
+    <p class="f5 tj">{{trailText}}</p>
```

Comment:
I'd recommend losing the `.tj` because I think it makes the word spacing a little more difficult to read, though this may well be a design decision on your part to emulate the style of a newspaper column!